cidata.py

```diff
@@ -0,0 +1,367 @@
+    podnet_b = f'{ipv6_subnet.split("/")[0]}10:0:3'
+
+    # Get `podnet_a_enabled` and `podnet_b_enabled`
+    podnet_a_enabled = config.get('podnet_a_enabled', None)
+    if podnet_a_enabled is None:
+        return False, messages[3014]
+    podnet_b_enabled = config.get('podnet_b_enabled', None)
+    if podnet_a_enabled is None:
+        return False, messages[3015]
+
+    # First run on enabled PodNet
+    if podnet_a_enabled is True and podnet_b_enabled is False:
+        enabled = podnet_a
+        disabled = podnet_b
+    elif podnet_a_enabled is False and podnet_b_enabled is True:
+        enabled = podnet_b
+        disabled = podnet_a
+    elif podnet_a_enabled is True and podnet_b_enabled is True:
+        return False, messages[3016]
+    elif podnet_a_enabled is False and podnet_b_enabled is False:
+        return False, messages[3017]
+    else:
+        return False, messages[3018]
+
+    # define payloads
+    create_userdata_payload = "\n".join([
+        f'tee {domain_path} userdata <<EOF',
+        userdata,
+        "EOF"
+        ])
+
+    create_metadata_payload = "\n".join([
+        f'tee {domain_path} metadata <<EOF',
+        metadata,
+        "EOF"
+        ])
+
+    # call rcc comms_ssh on enabled PodNet
+    try:
+        create_metadata, stdout, stderr = comms_ssh(
+            host_ip=enabled,
+            payload=create_metadata_payload,
+            username='robot',
+        )
+    except CouldNotConnectException:
+        return False, messages[3021]
+
+    if create_metadata.exit_code != SUCCESS_CODE:
+        return False, messages[3022] + f'{create_metadata.exit_code}s.'
+
+    # call rcc comms_ssh on enabled PodNet
+    try:
+        create_userdata, stdout, stderr = comms_ssh(
+            host_ip=enabled,
+            payload=create_userdata_payload,
+            username='robot',
+        )
+    except CouldNotConnectException:
+        return False, messages[3023]
+
+    if create_userdata.exit_code != SUCCESS_CODE:
+        return False, messages[3024]  + f'{create_userdata.exit_code}s.'
+
+    # call rcc comms_ssh on disabled PodNet
+    try:
+        create_metadata, stdout, stderr = comms_ssh(
+            host_ip=disabled,
+            payload=create_metadata_payload,
+            username='robot',
+        )
+    except CouldNotConnectException:
+        return False, messages[3031]
+
+    if create_metadata.exit_code != SUCCESS_CODE:
+        return False, messages[3032] + f'{create_metadata.exit_code}s.'
+
+    # call rcc comms_ssh on disabled PodNet
+    try:
+        create_userdata, stdout, stderr = comms_ssh(
+            host_ip=disabled,
+            payload=create_userdata_payload,
+            username='robot',
+        )
+    except CouldNotConnectException:
+        return False, messages[3033]
+
+    if create_userdata.exit_code != SUCCESS_CODE:
+        return False, messages[3034]  + f'{create_userdata.exit_code}s.'
+
+
+    return True, messages[1000]
+
+
+def scrub(
```

Comment:
Discussion for tomorrow: 

cidata.scrub is removing the two files in the sent `domain_path` from PodNets in HA. 

[directorymain.scrub()](https://github.com/CloudCIX/primitives/blob/cc5d087010c13630d67a17b90d8940cca3e675d5/directorymain.py#L266) removes directory for sent path from PodNets in HA.

Both are very similar, could the objective be met by sending `{domain_path}` as path in `directorymain.scrub()` 

`cidata_scrub()` would only be called by a driver when a VM is scrubbed. The VM driver would still be required to call `directorymain.scrub()` to remove the path created for this VMs cidata files.